features/frameworks/TARGET_PSA/pal/pal_crypto_config.h

```diff
@@ -27,11 +27,7 @@
 #ifndef _PAL_CRYPTO_CONFIG_H_
 #define _PAL_CRYPTO_CONFIG_H_
 
-#if !defined(MBEDTLS_CONFIG_FILE)
-#include "config.h"
-#else
-#include MBEDTLS_CONFIG_FILE
-#endif
+#include "mbedtls/config.h"
```

Comment:
There are two changes here:
1. change default header file include path
2. remove an option to override this file from app sources

I'm fine with the first one, the second one is not clear to me.
What is the reason for removing the ability to use custom config file?

Reply:
part 2 is actually made inside part 1
mbedtls overwriting the file, we should not do this.